kani-compiler/src/codegen_cprover_gotoc/compiler_interface.rs

```diff
@@ -144,7 +146,11 @@ impl CodegenBackend for GotocCodegenBackend {
             None
         };
 
-        let metadata = KaniMetadata { proof_harnesses: gcx.proof_harnesses };
+        let metadata = KaniMetadata {
```

Comment:
Do we want a constructor for `KaniMetadata`?  Maybe not given that its a POD class.

Reply:
Right, a constructor doesn't make sense here.